mesonbuild/mintro.py

```diff
@@ -546,7 +546,6 @@ def run(options: argparse.Namespace) -> int:
             assert backend is not None
             intr = IntrospectionInterpreter(sourcedir, '', backend.name, visitors = [AstIDGenerator(), AstIndentationGenerator(), AstConditionLevel()])
             intr.analyze()
-
```

Comment:
That blank line removal is in unrelated code. You're not even touching this file otherwise.

Reply:
This totally unrelated line seems to still be here?